template/golang-http/Dockerfile

```diff
@@ -26,13 +26,16 @@ ENV CGO_ENABLED=${CGO_ENABLED}
 # Run a gofmt and exclude all vendored code.
 RUN test -z "$(gofmt -l $(find . -type f -name '*.go' -not -path "./vendor/*" -not -path "./function/vendor/*"))" || { echo "Run \"gofmt -s -w\" on your Golang code"; exit 1; }
 
+RUN sh ./modules-cleanup.sh
```

Comment:
found it, gotta do the shell script right before `test -z`

Reply:
for reference, see here old format file; https://github.com/openfaas/golang-http-template/pull/81/files#diff-cab65cb33a949d149a9235a974191f15e1b6c43367b5301c9fd77cc59c396897; this last tested one as working